modules/git/repo_commit.go

```diff
@@ -12,15 +12,20 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/go-git/go-git/v5/plumbing"
-	"github.com/go-git/go-git/v5/plumbing/object"
 	"github.com/mcuadros/go-version"
+	"gopkg.in/src-d/go-git.v4/plumbing"
```

Comment:
```suggestion
	"github.com/go-git/go-git/v5/plumbing"
```

Reply:
This is on 1.11 - are we using go-git v5 on 1.11?

Reply:
yes we are, to fix date timezone bug